media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="+alsa +jack"
+
+RDEPEND="alsa? ( media-libs/alsa-lib )
+        jack? ( virtual/jack )"
```

Comment:
Just to clarify, all dependencies are always necessary. It doesn't matter that most people would already have those deps installed. Not having them breaks package management in subtle ways (and of course also breaks things like Travis and other automated test systems which *don't* have those deps installed). Anything the build system for a package touches on the host system at all must be a dependency, except for stuff in @system which I think you're allowed to assume is there (at least tools-wise; libraries other than glibc I'd still include as deps, especially if sub-slots are involved).